source/codegen/metadata/nixnet/config.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+config = {
+    'api_version': '21.0.0',
+    'c_header': 'nixnet.h',
+    'c_function_prefix': 'nx',
+    'service_class_prefix': 'NiXnet',
+    'java_package': 'com.ni.grpc.xnet',
```

Comment:
Looks like we don't have consistency between drivers in how these namespaces are defined. Some of them have "ni" repeated in namespace (`com.ni.grpc.nidaqmx`) vs. some omit `ni` from last component. Same for other language namespaces as well. Let's change `xnet` to `nixnet` here to keep it consistent with C# namespace below.